doc/index.rst

```diff
@@ -5,15 +5,42 @@ Welcome to the DESI Data Model!
 The DESI Data Tree
 ------------------
 
-DESI data are grouped broadly by category, using environment variables to
-define the base directory under which files of that category are kept.
-These variables have a standard location relative to :envvar:`DESI_ROOT`, but code
-uses these variables so that one can swap out different input/output locations
-for testing.
+These pages define the directory structure and file formats for DESI
+data products relative to a root directory :envvar:`DESI_ROOT`.
+Each data release has its own :envvar:`DESI_ROOT`, e.g. the
+`Early Data Release <https://data.desi.lbl.gov/doc/releases/edr/>`_
+at https://data.desi.lbl.gov/public/edr.
+
+Data Directories
+================
+
+Directories under DESI_ROOT
+
+   * `spectro/data/NIGHT/EXPID/ <DESI_SPECTRO_DATA/NIGHT/EXPID/index.html>`_: Raw data 
+   * `spectro/redux/SPECPROD/ <DESI_SPECTRO_REDUX/SPECPROD/index.html>`_: Processed spectra, classifications, and redshifts 
+   * `target/ <DESI_TARGET/index.html>`_: Target selection and fiber assignment catalogs
+   * `vac/RELEASE/ <DESI_ROOT/vac/RELEASE/index.html>`_: Value Added Catalogs
+
+The following directories are more expert-level (e.g. pipeline calibration inputs)
+and are documented for DESI collaboration internal use and may not be included
+in data releases:
+
+   * `survey/ops/surveyops/ <DESI_SURVEYOPS/index.html>`_: Data files used for day-to-day survey operations
```

Comment:
I would not explicitly link to `index.html`. Just link to `index` instead.

Reply:
... unless you tried that already and there was a warning about it.

Reply:
Context: I tried linking to `index` instead of `index.html`, but that generated invalid html links (to index without any extension) without any sphinx build warnings either.  Alternately, dropping `index.html` altogether and just referencing the directory also works for deploying to a real webserver that can resolve a directory into its index.html file, but that was more of a pain for developing on my laptop where Safari will follow a file link to a local file, but follows a directory link to opening the Finder on that directory instead. So I ended up referencing index.html...

Reply:
OK, right, now I remember. When you have internal links like this that are *not* in a `toctree`, you need to use `:doc:` and then the text + link.  There are several examples already, if you just grep for `:doc:`.

Reply:
got it.  I updated to use the `:doc:` syntax.